lib/MumbleConnection.js

```diff
@@ -551,10 +550,10 @@ MumbleConnection.prototype._onUDPTunnel = function( data ) {
     sequence = sequence.value;
     var moreFrames = true;
     while( moreFrames && packet.length > 0 ) {
-
         // Audio frame header.
         var headerLength, frameLength, terminateAudio;
         var header;
+        type = type & 0x7; // Only use the last 3 bits, as all other bits are used for the target which we ignore by now
```

Comment:
Nevermind! The type is read earlier and by now it should already be correct so masking it for 3 bits won't actually change it.

The real problem is on line 537:
https://github.com/Prior99/node-mumble/blob/fixwhisper/lib/MumbleConnection.js#L537

Those lines assume that the `target = 0`

I'll throw a fix in.